inc/page_cache/namespace.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Altis\Cloud\Page_Cache;
+
+use function Altis\Cloud\get_config;
+
+function bootstrap() {
+	global $batcache;
+	$batcache = [];
+
+	// Ignore common query params.
+	ignore_query_params();
+
+	// Forward CloudFront cache headers.
+	handle_cloudfront_headers();
+}
+
+function ignore_query_params() {
+	global $batcache;
+	$batcache = $batcache ?? [];
+
+	$config = get_config();
+	$ignored_query_string_params = $config['cache']['ignored-query-string-params'] ?? [
+		'utm_campaign',
+		'utm_medium',
+		'utm_source',
+		'utm_content',
+		'fbclid',
+		'_ga',
+	];
+
+	$batcache['ignored_query_string_params'] = $ignored_query_string_params;
+}
+
+function handle_cloudfront_headers() {
+	global $batcache;
```

Comment:
I think rather than having these functions be changing the global, I'd have them maybe be `get_batcache_config_for_cloudfront_headers` type thing, then have  `bootstrap` or similar handle merging those configs into the batcache global. Otherwise, we have a bunch of nasty side effects functions in this NS.

Reply:
Yep that makes sense! That way we can add a guard to check if it's appropriate to run in a single location if it's called again for any reason.